zap/zap.gradle.kts

```diff
@@ -149,6 +149,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
     packageExcludes = listOf(
         // Deprecated in 2.10, removal part of 2.11 milestone
         "org.parosproxy.paros.extension.encoder",
+        "org.parosproxy.paros.extension.report",
         "org.zaproxy.zap.extension.encoder2")
 
```

Comment:
Not a blocker but maybe but report at the bottom? I believe the comment at 150 applies to both encoder lines doesn't it?

Reply:
Done